apps/niftest/niftest.cpp

```diff
@@ -129,40 +125,38 @@ bool parseOptions (int argc, char** argv, std::vector<std::string>& files)
 int main(int argc, char **argv)
 {
     std::vector<std::string> files;
-    if(!parseOptions (argc, argv, files))
+    if (!parseOptions(argc, argv, files))
         return 1;
 
-//     std::cout << "Reading Files" << std::endl;
-    for(std::vector<std::string>::const_iterator it=files.begin(); it!=files.end(); ++it)
+    // std::cout << "Reading Files" << std::endl;
+    for (auto name : files)
     {
-        std::string name = *it;
-
         try
         {
-            if(isNIF(name))
+            if (isNIF(name))
+            {
+                // std::cout << "Decoding: " << name << std::endl;
```

Comment:
I suppose that these comments are not really needed.

Reply:
Jup. I plan on taking a bit more time in this PR, no style changes, purely functional changes. I'll take care of stuff like above to. I don't want this PR to rot and now that we're branched, we can see about getting this into master.

Reply:
You can take a look at the #2903, which does pretty the same thing, but without formatting changes.